hsmodels/schemas/aggregations.py

```diff
@@ -233,7 +234,7 @@ class MultidimensionalMetadataIn(BaseAggregationMetadataIn):
         title="Spatial coverage",
         description="An object containing the geospatial coverage for the aggregation expressed as either a bounding box or point",
     )
-    spatial_reference: Union[MultidimensionalBoxSpatialReference, MultidimensionalPointSpatialReference] = Field(
+    spatial_reference: MultidimensionalBoxSpatialReference = Field(
```

Comment:
I'm not seeing why this change?

Reply:
When testing I was getting errors with one specific netcdf file. Initially I thought the use of `MultidimensionalPointSpatialReference` might be causing the errors since hydroshare doesn't support point type spatial coverage for netcdf. You can see here (https://github.com/hydroshare/hydroshare/blob/develop/hs_file_types/models/netcdf.py#L96) when we serialize the OriginalCoverage (same as spatial reference) model to xml, we only use the coverage type to be Box. That's the reason I removed it. However, the errors that I was getting with one netcdf file was not due to the `MultidimensionalPointSpatialReference`. I ran all the unit tests after this change and they all passed. 

It seems if I keep this change, then I need to remove the class definition `MultidimensionalPointSpatialReference` as well as any references to it.

Reply:
okay

Reply:
I have now cleaned up code related to `MultidimensionalPointSpatialReference`.